samples/python/uppercase/uppercase.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   image: sk8s/grpc-py:latest
   protocol: grpc
   input: greetings
+  output: devnull
```

Comment:
we need a better solution for no-output cases (shouldn't involve any actual publishing)

Reply:
Yes. This is unrelated to this PR, rather a requirement of the go sidecar (which unconditionally creates a Publisher for now)

Reply:
understood - let's create an issue for that so we know it's tracked

Reply:
https://github.com/ericbottard/function-sidecar/issues/1